src/main/webapp/view/discussion/discussion.js

```diff
@@ -147,6 +154,30 @@ function updateNewCommentTimestamp(timeMs) {
   newCommentTimestampMs = timeMs;
 }
 
+/**
+ * Returns an array of the `DiscussionComment`s within
+ * `TIME_TOLERANCE_MILLISECONDS` milliseconds to the `timestampMilliseconds`.
```

Comment:
Perhaps this can be rephrased to

Return an array of `DiscussionComment`s in the range of  `TIME_TOLERANCE_MILLISECONDS` milliseconds and `timestampMilliseconds`?

It's a bit unclear the way it currently is.

Reply:
I moved things around, let me know how it looks